ccd-definition/CaseEventToFields/createOrder.json

```diff
@@ -37,6 +37,19 @@
     "PageLabel": " ",
     "PageDisplayOrder": 3,
     "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "orderFurtherDirections",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "FurtherDirections",
+    "PageLabel": " ",
+    "PageDisplayOrder": 4,
+    "PageColumnNumber": 1,
```

Comment:
hmm actually if we have this with no PageShowCondition here it'll be there for every type of order. I don't think we want this page for Blank orders (C21) as there is already a page for C21 where users can enter free text details (2nd page). I think we want this additional directions page for other types of order except C21, so maybe a show condition to make sure the order isn't a BLANK_ORDER? will also have to update e2e; in e2e test first we make a C21, then we make a care order, so this page should only appear for the care order not the C21

Reply:
Think that it is now done as you have suggested